src/radiation/closure.hpp

```diff
@@ -79,7 +79,6 @@ class ClosureEdd {
 
   //-------------------------------------------------------------------------------------
   /// Calculate the momentum density flux P^i_j from J, \tilde H_i, and \tilde pi^{ij}.
-  KOKKOS_FUNCTION
```

Comment:
Why does this not need to be a `KOKKOS_FUNCTION`?

Reply:
Oh this must have gotten lost when I went through a fairly involved merge conflict. Re-added, thanks.